docs/framework/additional-apis/toc.md

```diff
@@ -2,3 +2,10 @@
 ## [s_isDebuggerCheckDisabledForTestPurposes Field](s-isdebuggercheckdisabledfortestpurposes-field.md)
 ## [DataMemberFieldEditor Class](datamemberfieldeditor-class.md)
 ## [DataMemberListEditor Class](datamemberlisteditor-class.md)
+## [s_ServicePointTable Field](../../../docs/framework/additional-apis/s_servicepointtable.md)
```

Comment:
perhaps we could add the namespace as entry to the TOC now without any topic attached. I think it would increase readability.